.config/hakari.toml

```diff
@@ -25,5 +25,8 @@ platforms = [
 
 workspace-hack-line-style = "workspace-dotted"
 
+[traversal-excludes]
+workspace-members = ["grafbase-hooks", "grafbase-hooks-derive"]
```

Comment:
So... The test hooks want to use the grafbase-hooks crate from the workspace for easier tests. Now, if they require the workspace hack, it means there's suddenly tokio in the dependency chain. Tokio doesn't compile nicely to wasm32-wasip1, so we should exclude these two crates from hakari.

Luckily the SDK and derive have very small amount of dependencies.

Reply:
Does it actually fail to compile? My understanding is that cargo hakari is for Cargo.lock, the deps won't end up being compiled, since they aren't used.

Reply:
Yeah. So the dependency for workspace hack starts compiling axum and tokio in the examples without this.

Reply:
And wasip1 target fails hard there.